pkg/controller/pagerdutyintegration/pagerdutyintegration_controller.go

```diff
@@ -159,6 +161,11 @@ func (r *ReconcilePagerDutyIntegration) Reconcile(request reconcile.Request) (re
 
 	r.reqLogger = log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	r.reqLogger.Info("Reconciling PagerDutyIntegration")
+	if len(os.Getenv("FEDRAMP")) > 0 {
```

Comment:
should check for zero length
```suggestion
	if len(os.Getenv("FEDRAMP")) == 0 {
```